nexus/types/src/deployment/blueprint_diff.rs

```diff
@@ -879,7 +879,9 @@ pub struct BpDiffPhysicalDisks<'a> {
 }
 
 impl<'a> BpDiffPhysicalDisks<'a> {
-    pub fn from_diff_summary(summary: &'a BlueprintDiffSummary<'a>) -> Self {
+    pub fn from_diff_summary<'b>(
+        summary: &'b BlueprintDiffSummary<'a>,
+    ) -> Self {
```

Comment:
This can just be `&BlueprintDiffSummary<'a>`, and similarly for the others -- will fix before landing.

Reply:
Done.